daml-assistant/src/DAML/Assistant/Command.hs

```diff
@@ -64,5 +64,60 @@ installParser = InstallOptions
     <*> iflag ActivateInstall "activate" mempty "Activate installed version of daml"
     <*> iflag ForceInstall "force" (short 'f') "Overwrite existing installation"
     <*> iflag QuietInstall "quiet" (short 'q') "Don't display installation messages"
+    <*> enableDisableSwitch SetPath True "set-path"
```

Comment:
@associahedron I went with your suggestion of adding flags both ways but it’s rather annoying to do in optparse-applicative so not entirely sure I like the complexity it adds. If we add it, we should probably be consistent and add it to all flags (in a separate PR).

Reply:
I'm sorry for the bikeshedding but this isn't exactly what I had in mind. Although I do like `--flag` and `--no-flag` syntax, I was thinking of converting all the flags where the "correct" default behavior isn't obvious (`--activate` in particular) to `--flag=yes`, `--flag=no`, and `--flag=auto`, with `auto` being the default, which should be easier to implement in optparse-applicative. (The downside is that you need to pass in `yes` instead of just saying `--flag`... but good error messages should mitigate the downside.)

`--add-path` is another case where the default behavior isn't obvious (which is why I brought it up in the first place). On windows, you want to add path by default, and on unix you don't.

You can keep this as is for now, or revert to something simpler, or go all the way and implement `yes,no,auto` for `--add-path`. At some point I'm going to have to change the default behavior of `--activate`, so I'll update that flag when the time comes.

Reply:
Sounds sensible, I’ll switch to `yes,no,auto`.